src/shared/components/date-bubble/index.js

```diff
@@ -1,4 +1,5 @@
 // Displays date bubble
+/* eslint-disable max-len */
```

Comment:
You could go one further and use `this.props.ISODate` or whatever the field is called, along with [`format`](https://date-fns.org/docs/format) and use the in-built tokens